### PR TITLE
Add support for Yaesu FT-70D

### PR DESCRIPTION
**Pull request by n1zzo**
Add a new target radio, Yaesu FT-70D, which is flashed via USB through the proprietary protocol of its Hitachi H8SX Microcontroller BootROM.
The new target accepts raw binaries only, therefore the firmware compatibility checks are stubbed.
Please review it and let me know if there is anything to change.

### CODE REVIEW

**Comment by v0l** [comment]
Nice! can you also dump `lnusb -vvv` for the radio?

**Comment by n1zzo** [comment]
Sure thing:
```
~ » lsusb -vvv -d 045b:0025

Bus 001 Device 049: ID 045b:0025 Hitachi, Ltd
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0         8
  idVendor           0x045b Hitachi, Ltd
  idProduct          0x0025
  bcdDevice            1.00
  iManufacturer           0
  iProduct                0
  iSerial                 0
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0020
    bNumInterfaces          1
    bConfigurationValue     1
    iConfiguration          0
    bmAttributes         0xc0
      Self Powered
    MaxPower              100mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           2
      bInterfaceClass         0
      bInterfaceSubClass      0
      bInterfaceProtocol      0
      iInterface              0
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x01  EP 1 OUT
        bmAttributes            2
          Transfer Type            Bulk
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               0
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            2
          Transfer Type            Bulk
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               0
can't get device qualifier: Resource temporarily unavailable
can't get debug descriptor: Resource temporarily unavailable
Device Status:     0x0001
  Self Powered
  ```

**Comment by v0l** [comment]
Closes #17

File: src/usb_radio_factory.cpp [hunk 1]
```diff
@@ -79,8 +81,18 @@ auto USBRadioFactory::ListDevices(const uint16_t &idx_offset) const -> const std
                         auto cdev = const_cast<libusb_device *>(devs[x]);
                         if (LIBUSB_SUCCESS == (err = libusb_open(cdev, &h)))
                         {
-                            auto mfg = GetDeviceString(desc.iManufacturer, h),
-                                 prd = GetDeviceString(desc.iProduct, h);
+                            std::wstring mfg, prd;
+                            //Yaesu FT-70D doesn't support string descriptors
+                            if (desc.idVendor == YaesuRadio::VID && desc.idProduct == YaesuRadio::PID)
```
Comment (v0l) [inline src/usb_radio_factory.cpp:86]: Not a fan of this, does it break if its missing?

Comment (n1zzo) [reply src/usb_radio_factory.cpp:86]: The radio returns `LIBUSB_ERROR_PIPE` when you query the device string, so either add this special handling, or just return an empty string when the device string query fails, and `radio_tool -l` will return just the USB pid and vid and no text.
This is the downside of using an MCU from 2004.

**Pull request closed without merging**